### PR TITLE
Release 6.0.6-11

### PR DESCRIPTION
**Pull request by yuvallevy2**

### CODE REVIEW

File: admission.bundle.yaml [hunk 1]
```diff
@@ -75,7 +75,7 @@ spec:
       serviceAccountName: redis-enterprise-admission
       containers:
       - name: admin
-        image: redislabs/operator-internal:6.0.6-6
+        image: redislabs/operator-internal:6.0.6-11
```
Comment (amiramm) [inline admission.bundle.yaml:78]:
```suggestion
        image: redislabs/operator:6.0.6-11
```

File: admission.bundle.yaml [hunk 2]
```diff
@@ -106,7 +106,7 @@ spec:
             scheme: HTTPS
       initContainers:
       - name: admin-init
-        image: redislabs/operator-internal:6.0.6-6
+        image: redislabs/operator-internal:6.0.6-11
```
Comment (amiramm) [inline admission.bundle.yaml:109]:
```suggestion
        image: redislabs/operator:6.0.6-11
```
Surprised we missed these last time, why did we have all these internals?

File: admission.openshift.bundle.yaml [hunk 3]
```diff
@@ -75,7 +75,7 @@ spec:
       serviceAccountName: redis-enterprise-admission
       containers:
       - name: admin
-        image: redislabs/operator-internal:6.0.6-6.rhel7
+        image: redislabs/operator-internal:6.0.6-11.rhel7
```
Comment (amiramm) [inline admission.openshift.bundle.yaml:78]:
```suggestion
        image: redislabs/operator:6.0.6-11.rhel7
```

File: admission.openshift.bundle.yaml [hunk 4]
```diff
@@ -106,7 +106,7 @@ spec:
             scheme: HTTPS
       initContainers:
       - name: admin-init
-        image: redislabs/operator-internal:6.0.6-6.rhel7
+        image: redislabs/operator-internal:6.0.6-11.rhel7
```
Comment (amiramm) [inline admission.openshift.bundle.yaml:109]:
```suggestion
        image: redislabs/operator:6.0.6-11.rhel7
```

File: admission/deployment.yaml [hunk 5]
```diff
@@ -14,7 +14,7 @@ spec:
       serviceAccountName: redis-enterprise-admission
       containers:
       - name: admin
-        image: redislabs/operator-internal:6.0.6-6
+        image: redislabs/operator-internal:6.0.6-11
```
Comment (amiramm) [inline admission/deployment.yaml:17]:
```suggestion
        image: redislabs/operator:6.0.6-11
```

File: admission/deployment.yaml [hunk 6]
```diff
@@ -45,7 +45,7 @@ spec:
             scheme: HTTPS
       initContainers:
       - name: admin-init
-        image: redislabs/operator-internal:6.0.6-6
+        image: redislabs/operator-internal:6.0.6-11
```
Comment (amiramm) [inline admission/deployment.yaml:48]:
```suggestion
        image: redislabs/operator:6.0.6-11
```

File: admission/deployment_openshift.yaml [hunk 7]
```diff
@@ -14,7 +14,7 @@ spec:
       serviceAccountName: redis-enterprise-admission
       containers:
       - name: admin
-        image: redislabs/operator-internal:6.0.6-6.rhel7
+        image: redislabs/operator-internal:6.0.6-11.rhel7
```
Comment (amiramm) [inline admission/deployment_openshift.yaml:17]:
```suggestion
        image: redislabs/operator:6.0.6-11.rhel7
```

File: admission/deployment_openshift.yaml [hunk 8]
```diff
@@ -45,7 +45,7 @@ spec:
             scheme: HTTPS
       initContainers:
       - name: admin-init
-        image: redislabs/operator-internal:6.0.6-6.rhel7
+        image: redislabs/operator-internal:6.0.6-11.rhel7
```
Comment (amiramm) [inline admission/deployment_openshift.yaml:48]:
```suggestion
        image: redislabs/operator:6.0.6-11.rhel7
```

File: topics.md [hunk 9]
```diff
@@ -315,7 +315,7 @@ The finalizer name is `finalizer.redisenterprisedatabases.app.redislabs.com`.
 When a user requests the deletion of REDB (for example by running `kubectl delete redb <name>`), the following happens:
 1. K8s API adds `DeletionTimestamp` to the REDB resource.
 2. The Operator notices the `DeletionTimestamp`, and sends delete request to the RS API.
-3. When RS API the approves delete request, the operator removes the REDB finalizer.
+3. When RS API approves delete request, the operator removes the REDB finalizer.
```
Comment (amiramm) [inline topics.md:318]:
```suggestion
3. When RS API approves the delete request, the operator removes the REDB finalizer.
```

File: README.md [hunk 10]
```diff
@@ -377,7 +377,7 @@ spec:
 The Operator automates and simplifies the upgrade process.  
 The Redis Enterprise Cluster Software, and the Redis Enterprise Operator for Kubernetes versions are tightly coupled and should be upgraded together.  
 It is recommended to use the bundle.yaml to upgrade, as it loads all the relevant CRD documents for this version. If the updated CRDs are not loaded, the operator might fail.
-There are two ways to upgrade - either set 'autoUpgradeRedisEnterprise' within the Redis Enterprise Cluster Spec to instruct the operator to automatically upgrade to the compatible version, or specify the correct Redis Enterprise image manually using the versionTag attribute. The Redis Enterprise Version compatible with this release is 6.0.6-35
+There are two ways to upgrade - either set 'autoUpgradeRedisEnterprise' within the Redis Enterprise Cluster Spec to instruct the operator to automatically upgrade to the compatible version, or specify the correct Redis Enterprise image manually using the versionTag attribute. The Redis Enterprise Version compatible with this release is 6.0.6-39
```
Comment (amiramm) [inline README.md:380]:
```suggestion
There are two ways to upgrade - either set 'autoUpgradeRedisEnterprise' within the Redis Enterprise Cluster Spec to instruct the operator to automatically upgrade to the compatible version, or specify the correct Redis Enterprise image manually using the versionTag attribute. The Redis Enterprise version compatible with this release is 6.0.6-39
```

File: admission.bundle.yaml (hunk 2 above)
Comment (yuvallevy2) [reply admission.bundle.yaml:109]: This file was left out on the list of files that are replaced, thank you for catching that.

**Pull request merged**